chromium_src/ios/chrome/browser/shared/model/prefs/pref_names.h

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_IOS_CHROME_BROWSER_SHARED_MODEL_PREFS_PREF_NAMES_H_
+#define BRAVE_CHROMIUM_SRC_IOS_CHROME_BROWSER_SHARED_MODEL_PREFS_PREF_NAMES_H_
+
+#include "src/ios/chrome/browser/shared/model/prefs/pref_names.h"  // IWYU pragma: export
+
+namespace prefs {
+// A boolean specifying whether HTTPS Upgrades are enabled.
+inline constexpr char kHttpsUpgradesEnabled[] = "ios.https_upgrades_enabled";
```

Comment:
we normally wouldn't put this kind of stuff in a chromium_src override, but it appears that we're doing that quite a bit already in ios. We shouldn't use chromium_src overrides as a replacement for brave code that should go under ios. So for instance above in `BraveRegisterBrowserStatePrefs` should really go in `ios/brave/browser/shared/model/prefs/browser_prefs.mm` and same here.